app/models/proceeding_type_scope.rb

```diff
@@ -0,0 +1,48 @@
+class ProceedingTypeScope < ApplicationRecord
+  validate :valid_proceeding_type_code,
+           :valid_service_level,
+           :valid_client_involvement_type,
+           :valid_scope_limitation,
+           :only_one_default_in_group
+
```

Comment:
should valid_proceeding_type_code, valid_service_level, valid_client_involvement_type, valid_scope_limitation and only_one_default_in_group be private?

Reply:
probably, yes.  I'll fix that